python/cac_tripplanner/destinations/views.py

```diff
@@ -130,3 +133,24 @@ def get(self, request, *args, **kwargs):
             results = results[:limit_int]
         data = serializers.serialize('json', results, fields=output_fields)
         return HttpResponse(data, 'application/json')
+
+class FeedEvents(View):
+    """ API endpoint for the FeedEvent model """
+
+    def get(self, request, *args, **kwargs):
+        """ GET 20 most recent feed events that are published
+
+        TODO: Additional filtering, dynamic limits?
```

Comment:
I can add more params here, but I'm not sure ATM what is actually necessary, since I think the display of these events on the map is pretty basic (i.e. just show the clickable dots on a map with popup)